src/services/coronaTrackerService.js

```diff
@@ -0,0 +1,40 @@
+import { EventTargetShim } from '../utils/EventTargetShim.js';
+
+const cache = {};
```

Comment:
The cache should be on the class instance (that's why they're classes). Then when/if we do add tests we don't have globals.